client/capital/index.tsx

```diff
@@ -118,20 +121,28 @@ const getRowsData = ( loans: CapitalLoan[] ) =>
 					? loan.amount
 					: loan.amount / 100,
 				display: clickable(
-					formatExplicitCurrency( loan.amount, loan.currency )
+					formatExplicitCurrency(
+						loan.amount,
+						loan.currency.toUpperCase()
+					)
 				),
 			},
 			fee_amount: {
 				value: isZeroDecimalCurrency( loan.currency )
 					? loan.fee_amount
 					: loan.fee_amount / 100,
 				display: clickable(
-					formatExplicitCurrency( loan.fee_amount, loan.currency )
+					formatExplicitCurrency(
+						loan.fee_amount,
+						loan.currency.toUpperCase()
+					)
 				),
 			},
 			withhold_rate: {
 				value: loan.withhold_rate,
-				display: clickable( loan.withhold_rate + '%' ),
+				display: clickable(
+					( loan.withhold_rate * 100 ).toPrecision( 2 ) + '%'
```

Comment:
We already have a loan that has a `10.5%` withhold rate. With this code (2 significant digits) it would be displayed as `10%`. Let's remove the `toPrecision(2)` and show the full number. Worst it can happen is if the withhold rate is something like `13.333333333%`.

```suggestion
					( loan.withhold_rate * 100 ) + '%'
```

Reply:
I'll go with `+( loan.withhold_rate * 100 ).toFixed( 2 ) + '%'` as it shows 2 decimal points without trailing zeroes.